README.md

```diff
@@ -546,6 +546,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Chris Moreton](https://github.com/chris-moreton)
   - [Chris Niedermayer](https://github.com/Chris-N)
   - [Chris Schubert](https://github.com/C-Schubert94)
+  -[Christine Belzie](https://github.com/CBID2)
```

Comment:
Please can you add a space between the bracket and hyphen @CBID2 so the file is formatted?
Thank you.

Reply:
Can you do it for me @CodingSpecies? I'm not near my computer at the moment.

Reply:
I made the changes @CodingSpecies